### ISSUE TITLE
Insert command only inserts to first architecture in universal binaries

### ISSUE THREAD

**Issue by jonpalmisc** [cxnder/ktool#14]
I've attempted to insert a load command as follows:
```sh
ktool insert --lc load --payload libExample.dylib --out TargetPatched Target
```
This succeeds in adding the load command to the first architecture in the universal binary, however, it does not insert it to the second. The ability to either;

- choose which architecture to add the load command for; or
- automatically insert the load command in both architectures

would be helpful.

**Comment by cxnder** [comment]
![image](https://user-images.githubusercontent.com/55725881/167371263-b86b4240-b0b3-40f7-8627-eb9f5dd2a808.png)
No idea when this was fixed but as of 963e41c it works.

**Comment by cxnder** [comment]
Fix published in 1.3.0
